engine/include/cubos/engine/render/mesh/mesh.hpp

```diff
@@ -18,5 +18,11 @@ namespace cubos::engine
 
         /// @brief First bucket in the vertex pool which is part of the mesh.
         RenderMeshPool::BucketId firstBucketId{};
+
+        /// @brief Base offset of the mesh.
+        glm::vec3 baseOffset{};
+
+        /// @brief Bounding box of the mesh.
+        glm::vec3 boundingBox{};
```

Comment:
What does 'bounding box' mean here? If it is half the size of the bounding box, I would rename this to 'halfSize' or something like that. If it is the size, maybe just 'size'.